Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Build hw-event-proxy binaries
+# FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
+FROM docker.io/openshift/origin-release:golang-1.15 AS go-builder
+ENV GO111MODULE=off
+ENV CGO_ENABLED=1
+ENV COMMON_GO_ARGS=-race
+ENV GOOS=linux
+ENV GOPATH=/go
+
+COPY ./scripts /scripts
+WORKDIR /go/src/github.com/redhat-cne/hw-event-proxy/hw-event-proxy
+COPY ./hw-event-proxy .
+RUN /scripts/build-go.sh
```

Comment:
Does this work ? scripts and hw-event-proxy are in a different location now

Reply:
Yes Line# 10 copied the scripts to root dir